test/scripts/429s.js

```diff
@@ -1,6 +1,8 @@
 const client = new window.GenesysCloudStreamingClient(window.scConfig);
 window.client = client;
 
+
+
```

Comment:
NIT: Extra spacing? Is it just to make things cleaner and have clear separation of things?